comp/core/log/impl/logger.go

```diff
@@ -70,5 +73,8 @@ func NewComponent(deps Requires) (Provides, error) {
 		return nil
 	}})
 
-	return Provides{Comp: l}, nil
+	return Provides{
+		Comp:          l,
+		FlareProvider: flareprovider.NewLogFlareProvider(deps.Config, l),
```

Comment:
Did you added the package `"github.com/DataDog/datadog-agent/comp/core/log/internal/flareprovider"` 

I see it being reference, but I do not see in the diff. 

Did you pushed?